wallet/createtx.go

```diff
@@ -1206,18 +1204,7 @@ func (w *Wallet) purchaseTickets(ctx context.Context, op errors.Op,
 		return nil, err
 	}
 
-	var stakeSubmissionPkScriptSize int
-
-	// The stake submission pkScript is tagged by an OP_SSTX.
-	switch req.VotingAddress.(type) {
-	case *stdaddr.AddressScriptHashV0:
-		stakeSubmissionPkScriptSize = txsizes.P2SHPkScriptSize + 1
-	case *stdaddr.AddressPubKeyHashEcdsaSecp256k1V0, PubKeyHashAddress, nil:
-		stakeSubmissionPkScriptSize = txsizes.P2PKHPkScriptSize + 1
-	default:
-		return nil, errors.E(op, errors.Invalid,
-			"ticket address must either be P2SH or P2PKH")
-	}
+	const stakeSubmissionPkScriptSize = txsizes.P2PKHPkScriptSize + 1
```

Comment:
Please double check this line.

My understanding is that in most/all current usage of this func, it is called with an empty `req.VotingAddress` which means the second case from the switch should be the one which becomes hard-coded here.